Makefile

```diff
@@ -1,63 +1,85 @@
 export GO15VENDOREXPERIMENT=1
 
-SHORT_NAME := deis-e2e
+SHORT_NAME := workflow-e2e
 
 SRC_PATH := /go/src/github.com/deis/workflow-e2e
-DEV_IMG := quay.io/deis/go-dev:0.9.1
-
-RUN_CMD := docker run --rm -e DEIS_ROUTER_SERVICE_HOST=${DEIS_ROUTER_SERVICE_HOST} -e DEIS_ROUTER_SERVICE_PORT=${DEIS_ROUTER_SERVICE_PORT} -v ${CURDIR}:${SRC_PATH} -w ${SRC_PATH} ${DEV_IMG}
-DEV_CMD := docker run --rm -e GO15VENDOREXPERIMENT=1 -v ${CURDIR}:${SRC_PATH} -w ${SRC_PATH} ${DEV_IMG}
-
-TEST_OPTS := -slowSpecThreshold=120.00 -noisyPendings=false
-PARALLEL_TEST_OPTS := ${TEST_OPTS} -p
 
 MUTABLE_VERSION ?= canary
 VERSION ?= git-$(shell git rev-parse --short HEAD)
 
+ifdef GINKGO_NODES
+export GINKO_NODES_ARG=-nodes=${GINKGO_NODES}
+else
+export GINKO_NODES_ARG=-p
+endif
+
+TEST_OPTS := -slowSpecThreshold=120.00 -noisyPendings=false ${GINKO_NODES_ARG}
+
 DEIS_REGISTRY ?= quay.io/
 IMAGE_PREFIX ?= deis
 IMAGE := ${DEIS_REGISTRY}${IMAGE_PREFIX}/${SHORT_NAME}:${VERSION}
 MUTABLE_IMAGE := ${DEIS_REGISTRY}${IMAGE_PREFIX}/${SHORT_NAME}:${MUTABLE_VERSION}
 
-.PHONY: bootstrap
+ifndef DEIS_CONTROLLER_URL
+ifdef DEIS_ROUTER_SERVICE_HOST
+export DEIS_CONTROLLER_URL=http://deis.${DEIS_ROUTER_SERVICE_HOST}.nip.io
+endif
+endif
```

Comment:
I had done it this way because I felt it was more transparent.  Speaking personally, if I don't know how to run something (like a test suite), the Makefile is where I look first.  That said, I don't mind relocating this to the test code itself if you think there's a strong argument to be made for it.